pkgs/dartpad_ui/lib/widgets.dart

```diff
@@ -110,26 +110,31 @@ class ReloadButton extends ActionButton {
   const ReloadButton({super.key, super.onPressed})
     : super(
         text: 'Reload',
-        icon: const Icon(Icons.refresh, color: Colors.black, size: 20),
+        icon: const Icon(Icons.flash_on, color: Colors.black, size: 20),
```

Comment:
Thanks for the screenshot! Getting an icon that's the same as used in our other tools would be nice.